src/styles/accidenz_commons.scss

```diff
@@ -4,3 +4,23 @@
   font-weight: 500;
   src: url("../fonts/AccidenzCommons.otf") format("opentype");
 }
+.size {
```

Comment:
These styles should be in the story, not in the main stylesheet. The main stylesheet is for users of the library to be able to use the font.